documentation/tutorials/setup.rst

```diff
@@ -0,0 +1,130 @@
+==========================
+  How to setup kworkflow
+==========================
+
+.. contents::
+   :depth: 1
+   :local:
+   :backlinks: none
+
+.. highlight:: console
+
+Downloading kw
+--------------
+First of all, let's download kw::
+
+  git clone https://github.com/kworkflow/kworkflow.git
+
+We recommend checking out the unstable branch, to get the latest version
+of software::
+
+  git checkout unstable
+
+Alternatively, you can check if kw is available via package manager. For
+example, on debian-based distributions, you can try::
+
+  sudo apt install kworkflow
+
+but keep in mind that it might not get you kw's latest version.
+
+Installing kw
+-------------
+First, ``cd`` into the repository you cloned::
+
+  cd kworkflow
+
+Then install kw::
+
+  ./setup.sh --install
+
+Actually, in this folder you can see three executables: ``run_tests.sh``,
+``setup.sh`` and ``kw``. The file ``run_tests.sh`` is used by kw's developers to
+run it's test suite. ``setup.sh`` is used to install kw to your system, like we
+just did. ``kw`` is kw's main executable, but it should not be called from this
```

Comment:
maybe a quick explanation of why it shouldn´t be run like this even though it is possible? maybe mention the lack of tab completion and how this use is more directed for developers